### PR TITLE
Pre api

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
Brief summary of changes if not sufficiently described by commit messages.

New docker-compose profile for API development prerequisites.

## ⚙️ Test Data and/or Report
One of the following should be included here:
* Reference to regression test included in code (preferred wherever reasonable)
* Attach test data here + outputs of tests

### CODE REVIEW

File: docker/scripts/registry-loader-waits-for-elasticsearch.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+# ------------------------------------------------------------------------------------------------
+# This script is used to wait for Elasticsearch to start before starting the Registry API service.
+# ------------------------------------------------------------------------------------------------
+
+echo "Waiting for Elasticsearch to launch..."  1>&2
+while ! curl --output /dev/null --silent --head --fail http://elasticsearch:9200; do
+  echo "waiting for elasticsearch"
```
Comment (nutjob4life) [inline docker/scripts/registry-loader-waits-for-elasticsearch.sh:39]: Diagnostics like this should go to stderr 👍 
```bash
echo "waiting for elasticsearch" 1>&2
```

File: docker/scripts/wait-for-elasticsearch.sh [hunk 2]
```diff
@@ -36,6 +36,7 @@
 
 echo "Waiting for Elasticsearch to launch..."  1>&2
 while ! curl --output /dev/null --silent --head --fail http://elasticsearch:9200; do
+  echo "waiting for elasticsearch"
```
Comment (nutjob4life) [inline docker/scripts/wait-for-elasticsearch.sh:39]: Ditto 👍

**Review by nutjob4life — APPROVED** [review]
👍

File: docker/docker-compose.yml [hunk 3]
```diff
@@ -46,18 +46,22 @@ services:
...
     image: ${REG_LOADER_IMAGE}
     environment:
       - ES_URL=${ES_URL}
       - RUN_TESTS=true
       - TEST_DATA_URL=${TEST_DATA_URL}
     networks:
       - pds
+    volumes:
+      - ./scripts/registry-loader-waits-for-elasticsearch.sh:/usr/local/bin/registry-loader-waits-for-elasticsearch.sh
+    #command: ["sleep", "300"]
```
Comment (ramesh-maddegoda) [inline docker/docker-compose.yml:59]: This unused comment can be removed

**Review by ramesh-maddegoda — APPROVED** [review]

File: docker/scripts/registry-loader-waits-for-elasticsearch.sh (hunk 1 above)
Comment (tloubrieu-jpl) [reply docker/scripts/registry-loader-waits-for-elasticsearch.sh:39]: It is a more an info/debug message, but I am good with having that in stderr

File: docker/docker-compose.yml (hunk 3 above)
Comment (tloubrieu-jpl) [reply docker/docker-compose.yml:59]: Ok, I removed it

**Pull request merged**